test/cluster/functionality.sh

```diff
@@ -95,8 +95,27 @@ for elasticsearch_pod in $( oc get pods --selector component="${OAL_ELASTICSEACH
 
 		for kibana_pod in $( oc get pods --selector component="${OAL_KIBANA_COMPONENT}"  -o jsonpath='{ .items[*].metadata.name }' ); do
 			os::log::info "Cheking for index ${index} with Kibana pod ${kibana_pod}..."
-			# As we're checking system log files, we need to use `sudo`
-			os::cmd::expect_success "sudo -E VERBOSE=true go run '${OS_O_A_L_DIR}/hack/testing/check-logs.go' '${kibana_pod}' '${elasticsearch_api}' '${index}' '${index_search_path}' '${query_size}' '${test_user}' '${test_token}' '${test_ip}'"
+			elasticsearch_data="$(
+				oc exec "${kibana_pod}" -- curl --silent \
+                                                --key /etc/kibana/keys/key   \
+                                                --cert /etc/kibana/keys/cert \
+                                                --cacert /etc/kibana/keys/ca \
+                                                --header "X-Forwarded-For: ${test_ip}"         \
+                                                --header "X-Proxy-Remote-User: ${test_user}"   \
```

Comment:
As mentioned before it would be good to break out this authz test from the log test, but that can be done separately.